src/db.ts

```diff
@@ -0,0 +1,32 @@
+import * as Sequelize from 'sequelize';
+import * as pg from 'pg';
+import * as fs from 'fs';
+import * as board from './models/board';
+
+pg.defaults.ssl = true;
+let config = {
+  DATABASE_URL: undefined,
+};
+
+if (fs.existsSync('../config.js')) {
+  config = require('../config');
+}
+
+const sequelize = new Sequelize(process.env.DATABASE_URL || config.DATABASE_URL, {
+  pool: {
+    max: 20,
+    min: 0,
+    acquire: 30000,
+    idle: 10000,
+  },
+});
+// Get config from disk
+const db = {
+  Sequelize,
+  sequelize,
+  board,
+}
+
+db.sequelize.sync(/*{ force: true }*/)
+
+export default db;
```

Comment:
[We should avoid `export default`.](https://basarat.gitbooks.io/typescript/content/docs/tips/defaultIsBad.html)

I can take a look at this tonight.

Reply:
This isn't supposed to be this way.. I must have hit undo one to many times.. moment..